sdk/core/src/main/AndroidManifest.xml

```diff
@@ -15,4 +15,16 @@
             <data android:scheme="http" />
         </intent>
     </queries>
+
+    <application>
```

Comment:
Should have no effect unless the developer also follows the instructions below to add it, i.e.
```
 val fileLogger = FileLogger(context)
 fileLogger.attach()
 ```